docs/Deploying/Configuration.md

```diff
@@ -2580,7 +2580,19 @@ packaging:
 
 #### Mobile device management (MDM)
 
-> MDM features are not ready for production and are currently in development. These features are disabled by default.
+> MDM features are not ready for production and are currently in beta. These features are disabled by default. To enable these features, set `FLEET_DEV_MDM_ENABLED=1` as an environment variable.
+
+##### apple_mdm_enable
+
+This is the second feature flag required to turn on MDM features. This feature flag must be set to `1` at the same time as when you set the certificate and keys for Apple Push Certificate server (APNs) and Apple Business Manager (ABM). Otherwise, the Fleet server won't start.
```

Comment:
Is the other feature flag documented on this page?

Reply:
It was hiding in this note! https://github.com/fleetdm/fleet/pull/10373/files#diff-ac2e116fdc020a592bafe47e45117d69ad8e4490e8b197b193b54267b193ca6fR2583

Great feedback. @lukeheath I added reference docs for this feature flag to make it clearer that you need to configure this flag: https://github.com/fleetdm/fleet/pull/10373/files#diff-ac2e116fdc020a592bafe47e45117d69ad8e4490e8b197b193b54267b193ca6fR2585